### PR TITLE
Fix output from drafter, write to file

### PR DESCRIPTION
**Pull request by rohitpaulk**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,7 +27,7 @@ latest: blueprint  \
 
 .PHONY: draft
 draft:
-	@drafter blueprint.api -o apidraft.json -f json -t ast && gulp
+	@drafter blueprint.api -f json -t ast > apidraft.json && gulp
```
Comment (Gufran) [inline Makefile:30]: I failed to understand the problem here.

**Comment by rohitpaulk** [comment]
drafter doesn't support the `-o` option, I was getting an error that said `apidraft.json` doesn't exist or something similar

**Comment by rohitpaulk** [comment]
Does the original version of code work for you? If so this might be a problem with my setup

**Comment by Gufran** [comment]
It works for me, I generated all the other docs with it.

**Comment by rohitpaulk** [comment]
I'll have a look at this when I have time then

**Comment by rohitpaulk** [comment]
Looks like I just had an old version of drafter running, which didn't support `-o`

screen shot 2016-09-08 at 9 10 18 pm

**Comment by Gufran** [comment]
Glad that you figure this out finally

**Pull request closed without merging**